core/execution/liquidation/engine.go

```diff
@@ -132,14 +128,20 @@ func (e *Engine) Update(cfg *types.LiquidationStrategy) {
 	e.cfg = cfg
 }
 
-func (e *Engine) OnTick(ctx context.Context, now time.Time) (*types.Order, error) {
+func (e *Engine) OnTick(ctx context.Context, now time.Time, midPrice *num.Uint) (*types.Order, error) {
 	if e.pos.open == 0 || e.as.InAuction() || e.nextStep.After(now) {
 		return nil, nil
 	}
-	minP, maxP, err := e.ml.ValidOrdersPriceRange()
-	if err != nil {
```

Comment:
Just like what `ValidOrdersPriceRange()` does, if there is no mid-price we should bail out and not do any liquidation bits.

Reply:
good point